hooks/after_prepare/010_add_platform_class.js

```diff
@@ -38,7 +38,7 @@ function addPlatformBodyTag(indexPath, platform) {
 
     } else {
       // add class attribute to the body tag
-      newBodyTag = bodyTag.replace('>', ' class="' + platformClass + ' ' + cordovaClass + '">');
+      newBodyTag = bodyTag.replace(/>/g, ' class="' + platformClass + ' ' + cordovaClass + '">');
```

Comment:
This doesn't look right. It works, but it looks confusing.
It only works because `bodyTag` is guaranteed to only contain on `>` (based on [the code][1] that is used to extract `bodyTag`), but (a) you have to look at other places to verify that the code here works as expected and (b) it is semantically inaccurate/confusing. It says "replace all `>`", while one would expect to only replace one. Both behave in the same way when there is only one `>`, but it is confusing to read imo.

How about we switch to a different implementation that does not trigger the warning?
Maybe something like:
```suggestion
      var closingBrackerIdx = bodyTag.indexOf('>');
      newBodyTag = bodyTag.slice(0, closingBracketIdx) + ' class="' + platformClass + ' ' + cordovaClass + bodyTag.slice(closingBracketIdx);
```

Alternatively, we should add a comment explaining why the code looks this way.

[1]: https://github.com/xlts-dev/angularjs-ionic-v1/blob/c553c973d85142e66f119aed2fe3db83775f5f2b/hooks/after_prepare/010_add_platform_class.js#L57

Reply:
You are right. The `bodyTag` contains only the first found of `>` and not the whole collection, due to the return of the first element of the array from method (findBodyTag)[https://github.com/xlts-dev/angularjs-ionic-v1/blob/c553c973d85142e66f119aed2fe3db83775f5f2b/hooks/after_prepare/010_add_platform_class.js#L57]. Therefore, we should replace only one tag and not all of them.

Updated